grails-plugin-databinding/src/main/groovy/org/grails/databinding/converters/Jsr310ConvertersConfiguration.groovy

```diff
@@ -321,6 +321,25 @@ class Jsr310ConvertersConfiguration {
         }
     }
 
+    @Bean
+    ValueConverter instantStringValueConverter() {
+        new ValueConverter() {
+            @Override
+            boolean canConvert(Object value) {
```

Comment:
If `convert` casts to `CharSequence` then `canConvert` should check if `value` is instanceof `CharSequence`

Reply:
Good point, "String" is an unnecessary restriction